src/components/themes/dark/theme.css

```diff
@@ -437,7 +430,6 @@ html {
 
 ::-webkit-scrollbar-thumb:horizontal,
 ::-webkit-scrollbar-thumb:vertical {
-    -webkit-border-radius: 2px;
```

Comment:
There were a few places where you removed this and at least one where you just removed the vendor prefix instead.

Reply:
I re-added the ones that were removed as generic properties.